runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -183,6 +184,20 @@ public ClientUpdateCheckMode convert(String v)
 			System.exit(0);
 		}
 
+		if (options.has("debug"))
+		{
+			final Logger logger = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+			logger.setLevel(Level.DEBUG);
+		}
+
+		final ClientLoader clientLoader = new ClientLoader(options.valueOf(updateMode));
+
+		new Thread(() ->
```

Comment:
This should maybe me moved after the Thread default uncaught exception handler is setup so that logging works?